src/main/java/com/criticalsoftware/Product.java

```diff
@@ -1,23 +1,32 @@
 package com.criticalsoftware;
+
 import org.bson.types.ObjectId;
-import java.awt.image.BufferedImage;
 
 public class Product {
 
     private ObjectId id;
+    private String name;
     private String description;
-    private BufferedImage photo;
+    private String photo;
     private String category;
-    private User user;
 
     public Product() {
     }
 
-    public Product(String description, BufferedImage photo, String category, User user) {
+    public Product(String name,String description, String photo, String category, User userDonnor) {
```

Comment:
Why do you have the input parameter "userDonnor" if you don't use it in the constructor?